store/sqlstore/team_store.go

```diff
@@ -1005,21 +983,8 @@ func (s SqlTeamStore) GetAllForExportAfter(limit int, afterId string) ([]*model.
 	return data, nil
 }
 
-// GetUserTeamIds get the team ids to which the user belongs to
+// GetUserTeamIds get the team ids to which the user belongs to. allowFromCache parameter is used by SqlTeamStore but LocalCacheLayer
```

Comment:
Probably would be more clear to say `allowFromCache parameter does not have any effect in this Store`, the store doesn't need to know about other stores existence, and maintain consistency in this kind of comments is really hard.

Reply:
I was wrong in the message, it must have been `allowFromCache parameter is not used by SqlTeamStore but LocalCacheLayer` but I'll change it to your suggestion

Reply:
Done :dancer: